ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/daemon-terminattr.j2

```diff
@@ -6,8 +6,8 @@
 
 | CV Compression | CloudVision Servers | VRF | Authentication | Smash Excludes | Ingest Exclude | Bypass AAA |
 | -------------- | ------------------- | --- | -------------- | -------------- | -------------- | ---------- |
-{%     for cluster_name in daemon_terminattr.clusters | arista.avd.natural_sort %}
-{%         set cluster = daemon_terminattr.clusters[cluster_name] %}
+{%     for cluster_name in daemon_terminattr.clusters | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%         set cluster = cluster_name %}
```

Comment:
You can delete this line and in the above for loop, use cluster instead of cluster_name.

Reply:
Done